client/src/App.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+
+const App = props => {
+  const [array, setArray] = useState();
+
+  useEffect(
```

Comment:
fantastic job again on continuing to use your react skills and practicing with hooks